docs/specification.md

```diff
@@ -153,6 +155,35 @@ To specify the rules to calculate the holidays per country, state, region is as
 - `<timezone_?>` is a string representing the timezone. See file `data/meta/latest.json` of [moment-timezone][]
 - `<rule_?>` is the rule for that holiday
 
+## Citations, Sources, and Attribution
+
+Be sure to cite the URL where you obtained the holiday information. This project has support for two types of sources: primary and secondary/Wikipedia. To cite a primary source, add a YAML comment near corresponding specification. For example, from US.yaml:
+
+```yaml
+# @source https://www.federalreserve.gov/aboutthefed/k8.htm
+substitutes 11-11 if sunday then next monday if saturday then previous friday:
+  substitute: true
+  name:
+    en: Veterans Day
+  type: bank
+  note: Federal Government offices are closed
+```
+
+For Wikipedia sources, it is important to use `@attrib` instead of `@source`. The `doc:attrib` NPM task consolidates these attributions into the LICENSE file to comply with Wikipedia's [Creative Commons ShareAlike licencing](https://creativecommons.org/licenses/by-sa/3.0/). Here is an example of using `@attrib`:
+
+```yaml
+# @attrib https://en.wikipedia.org/wiki/Public_holidays_in_Bahrain
+holidays:
+  BH:
+    names:
+      ar: مملكة البحرين
+      en: Bahrain
+```
+
+### Source Guidelines
+
+Primary sources are preferable to secondary sources. Examples of good primary sources include but are not limited to government legislation, news articles, and national bank publications. Please refrain from including sources from data aggregators as often times the licencing is too restrictive or unclear to be used by an open source project.
```

Comment:
Let me know if you think I should be more specific about what a holiday data aggregator is. I have sites like https://www.timeanddate.com/ in mind.

Reply:
From my point of view the clarification is fine. I would not name a specific site here. Maybe you could add   
> Please refrain from including sources from data aggregators __and websites which offer holiday data__ ...

Reply:
Yea, I agree about not calling out specific sites. I updated the wording based on your suggestion. Let me know what you think.